frontend/demo/component/details/details-basic.ts

```diff
@@ -2,15 +2,21 @@ import '../../init'; // hidden-full-source-line
 
 import { html, LitElement, customElement } from 'lit-element';
 import '@vaadin/vaadin-details/vaadin-details';
+import '@vaadin/vaadin-ordered-layout/vaadin-vertical-layout';
 
 // tag::snippet[]
 @customElement('details-basic')
 export class Example extends LitElement {
   render() {
     return html`
-      <vaadin-details>
-        <div slot="summary">Expandable Details</div>
-        Toggle using mouse, Enter and Space keys.
+      <vaadin-details opened>
+        <div slot="summary">Contact Information</div>
```

Comment:
```suggestion
        <div slot="summary">Contact information</div>
```
Not sure how it was in the original design doc, but we use sentence case for UI texts.

Reply:
Done.